src/generics/middleware/authenticator.js

```diff
@@ -352,7 +354,9 @@ module.exports = async function (req, res, next, token = '') {
 				}
 
 				// convert the types of items to string
-				orgDetails.data.related_orgs = orgDetails.data.related_orgs.map(String)
+				orgDetails.data.related_orgs = orgDetails.data.organizations.map((data) => {
```

Comment:
@borkarsaish65  write a common function to utils to do this

Reply:
done